README.md

```diff
@@ -0,0 +1,66 @@
+![Hyperledger Sawtooth](https://raw.githubusercontent.com/hyperledger/sawtooth-core/master/images/sawtooth_logo_light_blue-small.png)
+
+# Hyperledger Sawtooth Java SDK
+
+## Install Sawtooth SDK from Central Repository
+
+#### Build a Sawtooth transaction processor
+
+```xml
+    <dependency>
+        <groupId>org.hyperledger.sawtooth</group>
+        <artifactId>sawtooth-sdk-transaction-processor</artifactId>
+        <version>v0.1.1</version>
+    </dependency>
+```
+
+#### Sign and verify signatures of transaction and batch headers
+
+```xml
+    <dependency>
+        <groupId>org.hyperledger.sawtooth</groupId>
+        <artifactId>sawtooth-sdk-transaction-processor</artifactId>
```

Comment:
Is this supposed to be `sawtooth-sdk-signing`?

Reply:
Yes, Thanks.